### PR TITLE
 fix(#8498): remove target enable settings 

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

medic/cht-core#8498

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8498-remove-targets-enabled/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8498-remove-targets-enabled/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8498-remove-targets-enabled/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by garethbowen** [comment]
@jkuester Can I grab a review please? This removes an obsolete and confusing button.

File: admin/src/js/controllers/targets.js [hunk 1]
```diff
@@ -14,25 +13,6 @@ angular.module('controllers').controller('TargetsCtrl',
     $scope.loading = true;
     $scope.error = false;
 
-    const setEnabled = function(value) {
```
Comment (jkuester) [inline admin/src/js/controllers/targets.js:17]: In vain did I look for the existing tests associated with this functionality....  😭

**Review by jkuester — APPROVED** [review]
This functionality was gone before I even knew it existed!  🤣  LGTM!

lever

File: admin/src/js/controllers/targets.js (hunk 1 above)
Comment (garethbowen) [reply admin/src/js/controllers/targets.js:17]: Heh, yeah... this is my favourite way to increase test coverage

**Pull request merged**